src/bots/settings/subCommands/tests/setDevChannel.test.js

```diff
@@ -32,7 +32,7 @@ describe('setting dev channel', () => {
     expect(process.env['DEV_CHANNEL_ID']).toEqual('12321');
 
     expect(client.message.channel.send).toHaveBeenCalledWith(
-      'Updated dev channel to current one.'
+      'Updated dev channel to the current one.'
```

Comment:
Still feels like broken english (even though I wrote it originally, so only myself to blame).